app/controllers/shoppinglists_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class ShoppinglistsController < ApplicationController
+  def show
+    @shoppinglist = current_user.menu.recipes
```

Comment:
This will cause an error unless current_user has a menu. If the user can access this action without a menu, you might want to do something about that.

Reply:
when a user signs up a menu is created for him